### ISSUE TITLE
Extension Tag Popup garbled in write/compose window header

### ISSUE THREAD

**Issue by TheysR** [Betterbird/thunderbird-patches#236]
Since upgrade to BB 15.5.2-bb20 using extension Tag Buttons (v 2.4.1), the tab buttons now appear vertically stack against the right, taking most of the header space of the write/compose window. I have not been able to find it in TB, it seems it has to do with the innovations in BB. It is good in previous version (bb19)

![Screenshot 2023-12-14 175600](https://github.com/Betterbird/thunderbird-patches/assets/98752192/a25cc227-4d7f-4f4d-8dd2-024204cab860)

I do not use TB since some considerable time,any more, so I am not able to try to replicate this with Thunderbird. I apologize if problems with add-ons should not be included here.

**Comment by Betterbird** [comment]
It's an issue in the add-on. Here is what we wrote to the author:

---

[A user] alerted us to the fact that your add-on "Tag Popup" doesn't behave correctly in Betterbird. We've analysed the problem and the solution is the following simple code change:

```
// Not working in Betterbird.
// let composeContainer = rw.document.getElementById("MsgHeadersToolbar");
// composeContainer.appendChild(row);
// Working in both Thunderbird and Betterbird.
let formatToolbox = rw.document.getElementById("FormatToolbox");
formatToolbox.prepend(row);
```

The technical background is that the addressing area has a grid template layout and after adding our attachments container, the UI element of "Tag Popup" gets put into the wrong position.

---

Corrected add-on attached.

[tag-toolbar-fixed-for-BB.xpi.zip](https://github.com/Betterbird/thunderbird-patches/files/13678361/tag-toolbar-fixed-for-BB.xpi.zip)
